bridges/DealabsBridge.php

```diff
@@ -263,15 +344,15 @@ private function getPrix($deal)
 	 */
 	private function getLivraison($deal)
```

Comment:
As your bridge now supports international websites, I would rename this to ```getDelivery```. Same for the rest.

Reply:
Good idea ! I did not read the function name anymore, only the content :)